src/styles/blocks/header.scss

```diff
@@ -0,0 +1,79 @@
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  margin: 0 50px;
+
+  header {
+    display: block;
+    padding: 0 50px;
+  }
+
+  @media screen and (max-width: 1024px) {
+    .container {
+      width: 1024px;
+    }
+  }
+
+  .nav {
+    display: flex;
+  }
+
+  a {
```

Comment:
use classes as selectors instead of tags

Reply:
done)